src/eureka/S5_lightcurve_fitting/plots_s5.py

```diff
@@ -793,3 +797,198 @@ def plot_GP_components(lc, model, meta, fitter, isTitle=True):
         fig.savefig(meta.outputdir+fname, bbox_inches='tight', dpi=300)
         if not meta.hide_plots:
             plt.pause(0.2)
+
+
+def plot_fleck_star(lc, model, meta, fitter):
+    
+    # find number of spots
+    spotinds = [s for s in model.parameters.dict.keys() if 'spotrad' in s]   
+    counter = [s for s in spotinds if '_' in s]
+    nspots = len(spotinds) - len(counter)
```

Comment:
This bit of code (and the one below for `limbdarks`) is quite unintuitive and will be hard to maintain

Reply:
yeah I struggled on how to do this quickly. I need to know the number of spots and since the names of parameters can have things appended for multiple planets and multiple channels, I can't just count how many "spotrad"s are in the parameters and needed to account for that.

Reply:
Maybe just tweak or add comments so that the future versions of ourselves can remember what is going on here?